homeassistant/components/spotify/media_player.py

```diff
@@ -598,7 +598,7 @@ def library_payload():
                 {"name": item["name"], "type": item["type"], "uri": item["type"]}
             )
         )
-    return library_info
+    return BrowseMedia(**library_info)
```

Comment:
Don't we already have a PR for this ?

Reply:
Yeah, merged and all... I reverted it but it stays... 🤷‍♂️  Git...